example/src/lib.rs

```diff
@@ -5,7 +5,9 @@ use anyhow::Result;
 use futures::Stream;
 use std::io::Read;
 
-ffi_gen_macro::ffi_gen!("example/api.rsh");
+mod bindings;
+use bindings::api;
+// ffi_gen_macro::ffi_gen!("example/api.rsh");
```

Comment:
why are we not using the build-macro anymore?

Reply:
Having generated code at hand makes debugging easier. The build script generates code exactly like `ffi_gen_macro` crate, so they are equivalent.

Reply:
right and that is fine for local debugging. but if the recommended usage-pattern is to use the macro, the example use that, too - otherwise our CI might not notice that we broke something. I am also okay with moving to a different usage model (of only generating the API once in a while - I am not super fond of having that macro in there), but then let's document that usage in the Readme instead...  Either way the main way of usage and the example should match up.

Reply:
I've reverted that code.